### ISSUE TITLE
Stats - Provide more context about different aggregate stats in similar looking cards

### ISSUE THREAD

**Issue by mkevins** [wordpress-mobile/WordPress-Android#17450]
### Expected behavior

The views and visitors aggregation stats less ambiguous, especially where the aggregations are performed differently for similar looking cards.

### Actual behavior

Some cards show different numbers, but the underlying aggregations are deliberately different. This can result in confusion. E.g. see this issues: https://github.com/wordpress-mobile/WordPress-Android/issues/17211, https://github.com/wordpress-mobile/WordPress-Android/issues/17212

This issue is to consider enhancing the UI to make it more clear why these numbers would be different.

More discussion here (internal ref): pbArwn-5yF-p2#comment-6939

**tl;dr** weeks view (and overview) show "fixed week" stats, with the current week usually being "incomplete". The insights card shows the latest available complete week compared to the week prior to that, on a _rolling_ basis (i.e. week ending yesterday, and week ending 7 days prior to yesterday) to eliminate bias in the performance metric. This difference is not immediately obvious from the similar design of the respective UIs, and can lead to the impression that the stats are inconsistent or inaccurate.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
